### DISCUSSION TITLE
스터디 2기 첫 모임 (Onboarding)

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#292] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. 달레입니다 👋

50명 정원이 금방 차서 멤버 모집을 이틀만에 조기 마감하게 되었습니다. 취합된 신청자 명단은 [여기에서](https://github.com/DaleStudy/leetcode-study/discussions/209) 확인이 가능하십니다. 많은 관심에 성원에 감사드립니다. 🙇‍♂️ 

스터디 2기 첫 번째 화상 모임은 한국 시간으로 8월 11일(일) 오전 10시 디스코드 서버의 "리트코드 공부방" 음성 채널에서 열립니다. 첫 모임에서는 리더, 코치, 멤버 분들이 간단한 자기 소개를 하고, 앞으로 15주 동안 스터디가 어떻게 진행되는지에 대해서 제가 자세하게 안내해드릴 예정입니다. 첫 모임에 빠지시면 사실상 온보딩(On-boarding)이 이뤄지지 않기 때문에, 출석하시는 분들을 최종 멤버로 확정해서 본격적인 스터디 2기를 시작하도록 하겠습니다.

![Shot 2024-08-05 at 19 39 08@2x](https://github.com/user-attachments/assets/5e60753d-77d6-4721-a84e-58fc5dda3cc8)

첫 화상 모임에 대한 참석 여부 회신(RSVP)는 아래 디스코드 이벤트 링크를 통해서 하실 수 있습니다. 참석이 하실 분들은 "관심 있음(Interested)" 버튼을 클릭해주시면 됩니다.

RSVP 👉: https://discord.gg/3Xw3wXrK?event=1270047352207573003

![Shot 2024-08-05 at 20 08 31@2x](https://github.com/user-attachments/assets/dc732218-0c72-49a1-8c7d-b802dbc87503)

관련해서 문의가 있으신 분들은 이 공지에 댓글로 부탁드리겠습니다. 모두 이번 주 일요일에 얼굴을 뵐 수 있었으면 좋겠습니다. 감사합니다! 🤗

**Comment by DaleSeo** [comment]
혹시 위 RSVP 링크가 작동하지 않는다면 디스코드 서버에 들어오셔서 좌측 상단에 이벤트를 클릭하시기 바랍니다.

![Shot 2024-08-05 at 20 19 11@2x](https://github.com/user-attachments/assets/95d8f4cf-6abc-459f-8f4c-605e6689c4f4)

**Comment by JoyJaewon** [comment]
제가 이번주에 남편 동생 생일이라 가족모임 가봐야하는데 혹시 녹화본으로 받거나 온보딩 참여할수있는 다른 방법이 있을까요?
모임시간이 현지 시간으로 토요일 오후라 거의 매주 그시간에 약속이 있습니다 흑
안되면 아쉽지만 2기에서는 드랍하고 다음 기수를 노려보겠습니다..!😢

**Reply by DaleSeo** [reply]
@JoyJaewon 매주 그 시간에 약속이 있으시면 첫 모임 뿐만 아니라 지속적으로 스터디 모임에 참석이 어려우시겠네요. 모임 시간은 벌써 여러 번 안내를 드려서 이제와서 변경은 어려울 것 같아요. 그럼 아쉽지만 다음 기회를 노려보시죠!

**Reply by JoyJaewon** [reply]
네네 알겠습니다. 그럼 저는 명단에서 제외해주세요! 다음에 다시 뵙겠습니당!

**Comment by SamTheKorean** [comment]
@TonyKim9401 @GyeongChan-Jang @0-Chan @movie5 @jdalma @dongsunleej  @nogy21 @sun912 @wooseok123 @kamg2218 @1eokim @YuA83 @GUMUNYEONG @HaJunYoo @hyejjun @whewchews @highballplz @jin0106 @CodyMan0 @sirzzang @YoonJongok @gitsunmin @JEONGHWANMIN @Sunjae95 @HC-kang @jaejeong1 @kimyoung  @mangodm-web @hololol-101 @f-exuan21 @KyrieJ95 @naringst @iriany @obzva 

안녕하세요! 다름 아니라 스터디 신청자분들 중에 첫 모임 RSVP를 안하셔서 리마인드 차원에서 멘션드립니다 . 매주 참여가 필수는 안하지만 첫 모임에 온보 딩을 진행할 예정이라 첫 모임 참여 인원을 확정인원으로 보고 진행할 예정이니 가급적 참여 부탁드립니다! RSVP는 아래 링크에 관심 있음 클릭해주시면 됩니다.https://discord.gg/SfxHQvHq?event=1270047352207573003

**Comment by YoonJongok** [comment]
안녕하세요 DaleSeo님. 제가 폴란드에 거주 중인데 온보딩 시간이 현지 시간으로 세벽 세시더라구요. 혹시 가능하다면 녹화본을 받을 수 있을까요? 
만약에 제 여건이 스터디 조건에 맞지 않다고 하면 너무 아쉽지만 다른 분을 위해 빠지는게 맞을 것 같습니다. 미리 확인하고 여쭤봤어야 했는데 죄송합니다. 의견 부탁드립니다!

**Reply by DaleSeo** [reply]
안녕하세요 @YoonJongok 님, 온보딩을 미처 녹화를 하지 못하였네요. ㅠㅠ 그런데 저희 모임 시간에 폴란드 기준 세벽 세시면 현실적으로 스터디 활동이 어려우실 것 같습니다. 아쉽지만 참여자 명단에서 제외시켜드리도록 할께요. 관심 주셔서 감사합니다!

**Reply by YoonJongok** [reply]
넵 감사합니다!

**Comment by 1eokim** [comment]
달레님 안녕하세요, 제가 지금 여행중이라 공지 및 안내를 너무 늦게 확인했네요. 늦게라도 꼼꼼히 깃헙과 디스코드 글 읽어보니 첫 온보딩 미팅 참석이 중요했던것 같네요.. 아쉽지만 2기에서는 하차하고, 3기를 노려보도록 하겠습니다! 🥹🙇🏻‍♂️
